src/EntityToolBar.tsx

```diff
@@ -28,7 +28,7 @@ const EntityToolBar = (props: AppProps) => {
     const { classes, registryId, handleNew, handlePersist, setAuthorised, chooseRegistry } = props;
 
     const handleLogout = () => {
-        setAuthorised('false');
+        setAuthorised('');

```

Comment:
Prefer a constant like we'd have in java that explains wha and why we're setting empty here